packages/devextreme/js/ui/html_editor/ui.html_editor.js

```diff
@@ -137,11 +137,18 @@ const HtmlEditor = Editor.inherit({
         });
 
         this._renderSubmitElement();
+        this._markupInitialization = true;
         this.callBase();
-
+        this._markupInitialization = false;
         this._updateContainerMarkup();
     },
 
+    _renderValidationState() {
+        if(!this._markupInitialization) {
```

Comment:
cannot we just use `this._isReady` here?

Reply:
reworked